manager-bundle/src/DependencyInjection/ContaoManagerExtension.php

```diff
@@ -32,5 +40,27 @@ public function load(array $configs, ContainerBuilder $container): void
         $loader->load('commands.yml');
         $loader->load('listener.yml');
         $loader->load('services.yml');
+
+        $this->configureManagerUrlParameter($mergedConfig, $container);
+    }
+
+    /**
+     * Configure the manager url porameter by checking configuration or default paths.
+     */
+    protected function configureManagerUrlParameter(array $mergedConfig, ContainerBuilder $container): void
+    {
+        $managerPath = null;
+
+        if ($mergedConfig['manager_path']) {
+            $managerPath = $mergedConfig['manager_path'];
+        } else {
+            $projectDir = $container->getParameter('kernel.project_dir');
+
+            if (is_file($projectDir . '/web/contao-manager.phar.php')) {
```

Comment:
`web` should not be hardcoded here, right? I thought we have something like `contao.web_dir` @leofeyer ?

Reply:
Yes we do. Sorry @dmolineus I did not know about that until this week 🙈

Reply:
Good to know that this configuration exists. I'll adjust this PR. By the way, you should start using it in the manager bundle as well :see_no_evil:
https://github.com/contao/contao/blob/efbc2e760f9d685fc69ed7812c2ea35555389114/manager-bundle/src/Command/InstallWebDirCommand.php#L123

Reply:
Changed this PR to use `contao.web_dir`.